h2/src/main/scala/org/scalarelational/h2/H2Datastore.scala

```diff
@@ -18,40 +16,31 @@ import org.scalarelational.table.Table
 abstract class H2Datastore private() extends SQLDatastore with Logging {
   protected def this(mode: H2ConnectionMode = H2Memory(org.powerscala.Unique()),
                      username: String = "sa",
-                     password: String = "sa") = {
+                     password: String = "sa") {
     this()
     dbUsername := username
     dbPassword := password
     modeProperty := mode
   }
 
-  protected def this(dataSource: DataSource) = {
```

Comment:
I couldn't find any occurrences where this constructor was used. I thought it is redundant as ``dataSourceProperty`` is ``public``.